lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart

```diff
@@ -1696,38 +1696,48 @@ class MailboxDashBoardController extends ReloadableController with UserSettingPo
     dispatchAction(StartSearchEmailAction());
   }
 
-  Future<void> selectFromSearchFilter() async {
+  Future<void> selectFromSearchFilter(BuildContext context) async {
```

Comment:
```suggestion
  Future<void> selectFromSearchFilter(AppLocalizations appLocalizations) async {
```

Reply:
Avoid passing around `BuildContext`, when all that method need is `AppLocalizations` from the `BuildContext`

Reply:
We will need more than just AppLocalizations from BuildContext

Reply:
Fixed